resources/assets/js/models/lookupConstants.ts

```diff
@@ -129,11 +129,15 @@ export const LocationId = {
 
 export enum JobStatus {
   Draft = 1,
-  Review = 2,
-  Approved = 3,
-  Open = 4,
-  Closed = 5,
-  Complete = 6,
+  ReviewManager = 2,
```

Comment:
As chatted, we could update these to string literal enums:
```ts
ReviewManager = "review_manager"
```

Reply:
I was starting to think this would be more trouble than its worth, but I think I see how to get this to work now. Will finish this up tomorrow morning.

Reply:
I've pushed this up now.